tests/test_ko.py

```diff
@@ -201,3 +201,37 @@ def test_construction(self):
         assert study_uid == self._sm_image.StudyInstanceUID
         assert series_uid == self._sm_image.SeriesInstanceUID
         assert instance_uid == self._sm_image.SOPInstanceUID
+
+    def test_construction_from_dataset(self):
+        document = KeyObjectSelectionDocument(
+            evidence=self._evidence,
+            content=self._content,
+            series_instance_uid=UID(),
+            series_number=10,
+            sop_instance_uid=UID(),
+            instance_number=1,
+            manufacturer='MGH Computational Pathology',
+            institution_name='Massachusetts General Hospital',
+            institutional_department_name='Pathology'
+        )
+        assert isinstance(document, KeyObjectSelectionDocument)
+        assert isinstance(document.content, KeyObjectSelection)
+
+        test_document = KeyObjectSelectionDocument.from_dataset(document)
```

Comment:
That is maybe a bit too simplistic. I would suggest to serialize the `document` instance to a buffer and read it back out using the pydicom `dcmwrite()` and `dcmread()` methods, respectively (see `test_seg.py` for examples).

Reply:
Fixed in `0b52144`